app/views/architectures/welcome.html.erb

```diff
@@ -1,14 +1,16 @@
-<% title_actions link_to(_("New architecture"), new_architecture_path) %>
-<% title _("Architecture configuration") %>
-<div id="welcome">
+<% content_for(:title, _("Architectures")) %>
+ <div class="blank-slate-pf">
+  <div class="blank-slate-pf-icon">
+    <i class="fa fa-building-o"></i>
+  </div>
+  <h1><%= _('Architectures') %></h1>
   <p>
-  <%= _("Before you proceed to using Foreman you should provide information about one or more architectures.") %>
-  </p>
-
-  <p>
-  <%= _("Each entries represent a particular hardware architecture, most commonly <b>x86_64</b> or <b>i386</b>. Foreman also supports the Solaris operating system family, which includes <b>sparc</b> based systems.").html_safe %>
-  </p>
-  <p>
-  <%= _("Each architecture can also be associated with more than one operating system and a selector block is provided to allow you to select valid combinations.") %>
+    <%= _("Before you proceed to using Foreman you should provide information about one or more architectures.") %></br>
+    <%= _("Each entries represent a particular hardware architecture, most commonly <b>x86_64</b> or <b>i386</b>. Foreman also supports the Solaris operating system family, which includes <b>sparc</b> based systems.").html_safe %></br>
+    <%= _("Each architecture can also be associated with more than one operating system and a selector block is provided to allow you to select valid combinations.") %>
   </p>
+  <p><%= _('Learn more about this') %> <%= link_to _('on the documentation'), documentation_url("4.4.5Architectures")%></p>
+  <div class="blank-slate-pf-main-action">
+      <%= link_to(_("New architecture"), new_architecture_path, :class => "btn btn-primary btn-lg") %>
```

Comment:
This is outside the scope of this PR, but just wanted to comment that the face we use js to add the classes to all links is not good practice - js should deal with interactions, not with styling that should have been done by the server.